data_source_obmcs_core_console_history_data_test.go

```diff
@@ -38,7 +38,7 @@ func (s *CoreConsoleHistoryDataDatasourceTestSuite) SetupTest() {
     }
     data "baremetal_core_console_history_data" "s" {
       console_history_id = "${baremetal_core_console_history.t.id}"
-      length = 1
+      length = 10240
```

Comment:
I agree. Probably should add a validator. That's just what the API error told me the min was.